packages/javascript/bh-shared-ui/src/views/ZoneManagement/DetailsRoot.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright 2025 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+import { Skeleton } from '@mui/material';
```

Comment:
It is important to make note that, while not currently a high priority, there is a long term goal of replacing components from Material UI with our own Doodle UI implementation. In some cases, we may not yet have feature parity, but it's a good idea to periodically peruse our [Doodle UI](https://cuddly-adventure-n8m7jwv.pages.github.io/?path=/docs/components-skeleton--docs) catalogue when you find you have to add a new component based on Material UI. I wouldn't go as far as saying you need to swap this out for this story, but doing a fast-follow refactor to that effect might be prudent.